api/src/types.rs

```diff
@@ -251,14 +266,60 @@ pub struct BlockHeaderInfo {
 }
 
 impl BlockHeaderInfo {
-	pub fn from_header(block_header: &core::BlockHeader) -> BlockHeaderInfo{
+	pub fn from_header(block_header: &core::BlockHeader) -> BlockHeaderInfo {
 		BlockHeaderInfo {
 			hash: util::to_hex(block_header.hash().to_vec()),
 			previous: util::to_hex(block_header.previous.to_vec()),
 			height: block_header.height,
```

Comment:
I'd add pretty much everything else there is in the header. The roots, timestamp, difficulty, etc. All can be useful.